packages/react-search-ui-views/stories/MultiCheckboxFacet.stories.js

```diff
@@ -27,10 +27,82 @@ const baseProps = {
   values: ["two", "three"]
 };
 
-storiesOf("Facets: Multi-value", module)
+const rangeOptions = [
+  {
+    count: 1,
+    value: {
+      from: 1,
+      to: 10,
+      name: "The first option"
+    }
+  },
+  {
+    count: 11,
+    value: {
+      from: 11,
+      to: 20,
+      name: "The second option"
+    }
+  }
+];
+
+storiesOf("Facets/MultiCheckboxFacet", module)
```

Comment:
Renamed stories to use the "/" syntax which actually nests them in storybook.

<img width="1010" alt="storybook" src="https://user-images.githubusercontent.com/1427475/53357122-1d3d1580-38fb-11e9-9e1e-6bd50fc4169d.png">